### ISSUE TITLE
FP for oelint.var.licenseremotefile if using WORKDIR in path

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#134]
```
LIC_FILES_CHKSUM = "file://${WORKDIR}/MIT;md5=sjdjasdjhddh"
```
is giving a false positive
